src/core/rendering.js

```diff
@@ -796,23 +796,23 @@ p5.prototype.clearDepth = function(depth) {
  * function setup() {
  *   createCanvas(100, 100);
  *
- *   background(200);
+ *   background(180);
  *
  *   // Set the blend mode.
- *   blendMode(ADD);
+ *   blendMode(HARD_LIGHT);
  *
  *   // Style the lines.
  *   strokeWeight(30);
  *
- *   // Draw the blue line.
- *   stroke('blue');
+ *   // Draw the first line.
+ *   stroke('#25b7ff');
```

Comment:
Throughout all the examples provided, the colors used are '#1a85ff' and '#d41159'.